spatialprofilingtoolbox/db/squidpy_metrics.py

```diff
@@ -42,39 +36,31 @@ def create_and_transcribe_squidpy_features(
             create_and_transcribe_one_sample(
                 sample,
                 df,
-                channel_symbols_by_column_name,
                 feature_uploader,
             )
 
 
 def create_and_transcribe_one_sample(
     sample: str,
     df: DataFrame,
-    channel_symbols_by_column_name: dict[str, str],
     feature_uploader: ADIFeaturesUploader,
 ) -> None:
-    for column, symbol in channel_symbols_by_column_name.items():
-        criteria = PhenotypeCriteria(positive_markers=[column], negative_markers=[])
-        value = compute_squidpy_metric_for_one_sample(df, [criteria], 'spatial autocorrelation')
-        if value is None:
-            continue
-        feature_uploader.stage_feature_value((symbol,), sample, value)
+    for column in df.columns:
+        if column.startswith('C '):
+            symbol = column[2:]
+            criteria = PhenotypeCriteria(positive_markers=[symbol], negative_markers=[])
+            value = compute_squidpy_metric_for_one_sample(df, [criteria], 'spatial autocorrelation')
+            if value is None:
+                continue
+            feature_uploader.stage_feature_value((symbol,), sample, value)
```

Comment:
Both old and new versions of this function create and transcribe spatial autocorrelation for only channels (simple phenotypes). Do we want to do autocorrelation for (complex) phenotypes as well?

(It might require some another input parameter since the columns will have the complex phenotype name, but not its signature, so that'll have to be fetched from elsewhere.)

Reply:
I don't think it's necessary to do this, since the primary use case is the ondemand anyway.